src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -329,6 +329,21 @@ public static bool IsPortablePdb (string filename)
 				return false;
 			}
 		}
+
+		internal static string QuoteFileName (string fileName)
+		{
+			var builder = new CommandLineBuilder ();
+			builder.AppendFileNameIfNotNull (fileName);
+			return builder.ToString ();
+		}
+
+		internal static string GetShortPath (string path)
+		{
+			if (Environment.OSVersion.Platform != PlatformID.Win32NT)
+				return QuoteFileName (path);
+			var shortPath = KernelEx.GetShortPathName (Path.GetDirectoryName (path));
+			return Path.Combine (shortPath, Path.GetFileName (path));
```

Comment:
This should be:

```csharp
return QuoteFileName (Path.Combine (shortPath, Path.GetFileName (path)));
```

Reply:
@jonpryor I don't want to make this change in this PR.. It will impact on the AoT task and we don't really want to change that until we get the mono response file working.